### ISSUE TITLE
Improve formula creation command

### ISSUE THREAD

**Issue by fabianofernandeszup** [ZupIT/ritchie-cli#653]
Add questions to fill in the help.json and decription fields of readme and the inputs of config.json.

```
$ rit create formula
? Enter the new formula command:  rit github publish release
? Choose the language: shell
? Select a formula workspace: default
? Formula description (default blank): Formula to publish a new release on a github project
```

**Comment by GuillaumeFalourd** [comment]
I remember the first version of the formula creation was like that.

Won't it be too much to config inputs this way with all the [current possibilities](https://docs.ritchiecli.io/tutorials/formulas/implement-a-formula) ?

We can currently config between 3 and 21 fields for each input. Seems a lot to me.

**Comment by fabianofernandeszup** [comment]
The configuration of the input is optional, if he wants to use a text editor for this.

**Comment by GuillaumeFalourd** [comment]
> The configuration of the input is optional, if he wants to use a text editor for this.

Yes, we did it like this last time as well.
It's good to have the option to configure them through the command line.
My doubts are more related to usability: Will the developer really use this option when creating formulas?

I think we should create a form to get feedbacks about this feature first, as it will add a lot of dependencies each time we'll add new inputs configuration options. And we'll have to think the formulas templates structure as well.

**Comment by henriquemoraeszup** [comment]
Resolved in a more automated manner by #689
